sample_app/lorawan_aio/_index.md

```diff
@@ -0,0 +1,535 @@
+# LoRaWAN® with Azure IoT operations
+
+## Introduction
+
+According to Lora Alliance, “LoRaWAN® is a Low Power, Wide Area (LPWA) networking protocol designed to wirelessly connect battery operated ‘things’ to the internet in regional, national or global networks, and targets key Internet of Things (IoT) requirements such as bi-directional communication, end-to-end security, mobility and localization services.”
+In simple terms, LoRaWAN® allows low-cost and low-power communication between devices and gateways within the range of physical buildings.
+Advantages of LoRaWAN®:
+• Ideal for single-building uses/applications
+• Easy to set up and manage your personal network
+• LoRa devices work without strain even when in motion
+• Devices using LoRa tech have extended/long battery lives
+• Supports bidirectionality such as command-and-control functionality
+
+We are looking at more and more devices using LaRaWAN as their communication protocol. For those systems that are based on LoRaWAN®, how can we incorporate with Azure IoT Operations? To this answer this question, we are going to implement a solution that takes data from LoRaWAN® based system and then pushes it to Azure IoT Operations. This is done using a custom broker that understands LoRaWAN® communication data and talks to Azure IoT operations using MQTT and portforwarding using a broker listener.
+
+## Architecture
+
+![alt text](./images/architecture.png)
+
+## Solution overview
+
+A typical LoRaWAN® based architecture comprises of:
+
+- LoRa Devices
+- LoRaWAN® Gateway
+- A network server [optional]
+
+The UDP data originates from a LoRa device in the field is sent to LoRaWAN® Gateways that are service multiple LoRa devices. Once the data reaches LoRaWAN® Gateways, it is forwarded to a network server using a Gateway Bridge that is part of the LoRaWAN® Gateway. Typically a network server would have applications registred that would process the incoming UDP data fro upstream systems.
+
+In our case we will be using LWN Simulator to simulate LoRa devices and LoRaWAN® Gateways. Instead of using network server, we will be implementing our own custom LoRaWAN® Broker that will receive UDP data from LoRaWAN® Gateway simulator as part of LWN Simulator using Gateway Bridge.
+
+After receiving LoRaWAN® UDP data, the custom LoRaWAN® Broker, would forward data to Azure IoT Operations using MQTT broker. This is how the data from simulated LoRa device would make it to Azure IoT operations through simulated LoRaWAN® Gateway and custom LoRaWAN® Broker.
+
+## System requirements
+
+### VM that can run Azure IoT operations
+
+For our example, we are going to be using Ubuntu VM to build the solution.
+
+## Prerequisites
+
+### Git
```

Comment:
I'd avoid using title/subtitles -> List them (maybe bold it) and then just explain it, but dont use headers

Reply:
Added the list instead of heading for 'Git'. This is part of the commit: https://github.com/nabeelmsft/arc_jumpstart_drops/commit/904d3ae0b5c741939cc41d86c9b27504834f4816
Quick note, when I tried to use bold, the markdown lint error [MD036](https://github.com/DavidAnson/markdownlint/blob/v0.34.0/doc/md036.md).